lib/openstack/openstack_handle/handle.rb

```diff
@@ -53,7 +55,13 @@ def self.raw_connect(username, password, auth_url, service = "Compute", extra_op
       # throw an error trying to build an connection error message.
       opts[:openstack_service_type] = ["object-store"] if service == "Storage"
 
-      Fog.const_get(service).new(opts)
+      if service == "Planning"
+        # Special behaviour for Planning service Tuskar, since it is OpenStack specific service, there is no
+        # Fog::Planning module, only Fog::OpenStack::Planning
+        Fog::Openstack.const_get(service).new(opts)
```

Comment:
Does it make sense to just do this for all the services?  Since this is openstack-specific code...

I haven't looked at Fog recently enough to know if doing `Fog::Openstack.const_get(service).new(opts)` will work for everything.

cc @Fryguy